AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -555,8 +577,8 @@ namespace CGAL {
     AABBTraits m_traits;
 		// set of input primitives
 		Primitives m_primitives;
-		// single root node
-		Node* m_p_root_node;
```

Comment:
Using raw pointer to manage an array. Also it's a little misleading as it's a pointer to an array not a single node.